plugins/invisible-typing/src/Icon.tsx

```diff
@@ -17,43 +17,41 @@ export const Icon = (props: { type?: { analyticsName: string } }) => {
   const channelWise = cfg.get("button", true) ? cfg.get("channelWise", true) : false;
   const channelsList = cfg.get("channels", { [channelId]: globalInvisible });
   const [enabled, setEnabled] = React.useState(
-    channelWise ? channelsList[channelId] : globalInvisible,
+    channelWise ? channelsList[channelId] ?? globalInvisible : globalInvisible,
   );
   return (
-    <>
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Simplified component structure by removing unnecessary fragment.

The removal of the React fragment in favor of a div directly keyed by the enabled state simplifies the component's structure without altering its functionality.

```suggestion
<div key={`${enabled}`}>
```